modules/3d/include/opencv2/3d.hpp

```diff
@@ -2805,6 +2805,86 @@ CV_EXPORTS_W void loadMesh(const String &filename, OutputArray vertices, OutputA
 */
 CV_EXPORTS_W void saveMesh(const String &filename, InputArray vertices, InputArray normals, InputArrayOfArrays indices);
 
+//! Triangle fill settings
+enum TriangleShadingType
+{
+    White  = 0, //!< a white color is used for the whole triangle
+    Flat   = 1, //!< a color of 1st vertex of each triangle is used
+    Shaded = 2  //!< a color is interpolated between 3 vertices with perspective correction
+};
+
+//! Face culling settings: what faces are drawn after face culling
+enum TriangleCullingMode
+{
+    None = 0, //!< all faces are drawn, no culling is actually performed
```

Comment:
constants names:
- must be uppercase
- must have prefix to avoid conflicts

Reply:
Fixed